stock_inventory_adjustment/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

Comment:
As a convention, no need to have copyright and license info in __init__.py files where only import is done.  This is just for simplicity.